### ISSUE TITLE
Ember do not send Content-Type for request without body

### ISSUE THREAD

**Issue by neomerx** [neomerx/json-api#128]
Confirmed for GET, not sure about DELETE https://github.com/emberjs/data/issues/3523

The json-api spec do not allow such exceptions though the header is not sent and request can't pass headers checker. It's proposed to make it configurable and skip such check for requests like GET.

**Comment by mphasize** [comment]
Also related: https://github.com/json-api/json-api/issues/968

If there's a `Content-Type` in GET requests, it will apparently trigger a CORS preflight request.

**Comment by neomerx** [comment]
@mphasize I've made many changes in #125 and then switched to this one. From code it looks like if no `Content-Type` header is added to request then the lib will assume it has JSON-API media type by default. I've checked it with demo app and it seems to be correct (note `Content-Type` is absent)
![image](https://cloud.githubusercontent.com/assets/10420662/14718612/613dbd36-0800-11e6-866b-c1bd5f10a00e.png)

I'm closing this issue. If it appears again it should be looked more closely which headers are sent as just absence of `Content-Type` do not cause any errors.
